components/builder-originsrv/src/migrations/origin_public_keys.rs

```diff
@@ -83,11 +83,15 @@ pub fn migrate(migrator: &mut Migrator) -> Result<()> {
                         RETURN;
                     END
                     $$ LANGUAGE plpgsql STABLE"#)?;
-
-    migrator.migrate("originsrv",
-                     r#"ALTER TABLE origin_public_keys
+    migrator
+        .migrate("originsrv",
+                 r#"ALTER TABLE origin_public_keys
                         DROP CONSTRAINT IF EXISTS
                           origin_public_keys_full_name_key"#)?;
-
+    migrator
```

Comment:
I feel hesitant about adding queries that directly manipulate data (as opposed to schemas) to the migrator.  That was a constant source of problems with Rails apps because the data's always changing and it's inevitable that that query will fail and error when run against the wrong data set.